src/dev/i18n/integrate_locale_files.test.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import path from 'path';
+
+import { verifyMessages, integrateLocaleFiles } from './integrate_locale_files';
+import { normalizePath } from './utils';
+
+const localePath = path.resolve(__dirname, '__fixtures__', 'integrate_locale_files', 'fr.json');
+
+jest.mock('./extract_default_translations.js', () => ({
+  getDefaultMessagesMap: () => {
+    return new Map([
+      ['plugin-1.message-id-1', 'Message text 1'],
+      ['plugin-1.message-id-2', 'Message text 2'],
+      ['plugin-2.message-id', 'Message text'],
+    ]);
+  },
+}));
+
+const { getDefaultMessagesMap } = require('./extract_default_translations.js');
```

Comment:
__nit:__ no need to do this, just do smth like this:
```js
const mockDefaultMessagesMap = new Map([
  ['plugin-1.message-id-1', 'Message text 1'],
  ['plugin-1.message-id-2', 'Message text 2'],
  ['plugin-2.message-id', 'Message text'],
]);

jest.mock('./extract_default_translations.js', () => ({
  getDefaultMessagesMap: () => mockDefaultMessagesMap,
}));
```

And use `mockDefaultMessagesMap` where needed.